cypress/integration/explore.spec.js

```diff
@@ -75,4 +84,72 @@ context('Actions', () => {
   });
+
+  it('adds file types as a filter', () => {
+    cy.visit('/');
+    cy.get('[data-testid=search-for-resource-input]').type(search);
+    cy.get('[data-testid=search-for-resource-submit]').click();
+    cy.get('[data-testid=expand-filtering-options]').click();
+    cy.get(
+      `[data-testid=resource-type-filtering-checkbox-label-${ResourceFeatureTypes.audio.toString().toLowerCase()}]`
+    ).click();
+    cy.get(
+      `[data-testid=resource-type-filtering-checkbox-label-${ResourceFeatureTypes.video.toString().toLowerCase()}]`
+    ).click();
+    cy.get(
+      `[data-testid=resource-type-filtering-checkbox-label-${ResourceFeatureTypes.document.toString().toLowerCase()}]`
+    ).click();
+    cy.location().should((loc) => {
+      expect(loc.search).to.eq(
+        `?${SearchParameters.query}=${search}&${SearchParameters.resourceType}=${ResourceFeatureTypes.audio}&${SearchParameters.resourceType}=${ResourceFeatureTypes.video}&${SearchParameters.resourceType}=${ResourceFeatureTypes.document}`
+      );
+    });
+  });
+
+  it('can remove one or all resource types filters', () => {
+    cy.visit('/');
+    cy.get('[data-testid=search-for-resource-input]').type(search);
+    cy.get('[data-testid=search-for-resource-submit]').click();
+    cy.get('[data-testid=expand-filtering-options]').click();
+    cy.get(
+      `[data-testid=resource-type-filtering-checkbox-label-${ResourceFeatureTypes.audio.toString().toLowerCase()}]`
+    ).click();
+    cy.get(
+      `[data-testid=resource-type-filtering-checkbox-label-${ResourceFeatureTypes.video.toString().toLowerCase()}]`
+    ).click();
+    cy.location().should((loc) => {
+      expect(loc.search).to.eq(
+        `?${SearchParameters.query}=${search}&${SearchParameters.resourceType}=${ResourceFeatureTypes.audio}&${SearchParameters.resourceType}=${ResourceFeatureTypes.video}`
+      );
+    });
+    cy.get(
+      `[data-testid=resource-type-filtering-checkbox-label-${ResourceFeatureTypes.audio.toString().toLowerCase()}]`
+    ).click();
+    cy.location().should((loc) => {
+      expect(loc.search).to.eq(
+        `?${SearchParameters.query}=${search}&${SearchParameters.resourceType}=${ResourceFeatureTypes.video}`
+      );
+    });
+    cy.get(
+      `[data-testid=resource-type-filtering-checkbox-label-${ResourceFeatureTypes.video.toString().toLowerCase()}]`
+    ).click();
+    cy.location().should((loc) => {
+      expect(loc.search).to.eq(`?${SearchParameters.query}=${search}`);
+    });
+  });
+
+  it('can detect file types filters in the url', () => {
+    cy.visit(
+      `/?${SearchParameters.query}=${search}&${SearchParameters.resourceType}=${ResourceFeatureTypes.audio}&${SearchParameters.resourceType}=${ResourceFeatureTypes.video}`
+    );
+    cy.get('[data-testid=expand-filtering-options]').click();
+    cy.get(`[data-testid=resource-type-filtering-checkbox-${ResourceFeatureTypes.audio.toString().toLowerCase()}]`)
+      .children('span')
+      .children('input')
```

Comment:
kan skrives rett i css-selectoren `cy.get(`[data-testid=resource-type-filtering-checkbox-${ResourceFeatureTypes.audio.toString().toLowerCase()} input]`)`

Reply:
Får feilmelding: 

![image](https://user-images.githubusercontent.com/44363838/110797943-f2238580-8279-11eb-8019-21635996a6ce.png)

Reply:
skrivefeil. må bak` ]`

Reply:
get([data-testid=resource-type-filtering-checkbox-${ResourceFeatureTypes.audio.toString().toLowerCase()}]).get('input') fungerer ikke som tiltenkt. gikk tilbake til children metoden

Reply:
testet med 
```
    cy.get(
      `[data-testid=resource-type-filtering-checkbox-${ResourceFeatureTypes.audio.toString().toLowerCase()}] input`
    ).should('be.checked');
```
funket greit

Reply:
Men ikke variant av  cy.get(
      `[data-testid=resource-type-filtering-checkbox-${ResourceFeatureTypes.audio.toString().toLowerCase()}] input`
    ).should('not.be.checked');
    
    Så da ble jeg usikker på om det "be.checked" var en eller annen form for falsk positiv.

Reply:
Fikset